### PR TITLE
Fixes #657: Display a message when no skill usage data is present

### LINKED ISSUE #657: Default message when the skill has no usage
**Actual Behaviour**
It doesn't make sense to render a graph when the skill hasn't been used.

**Expected Behaviour**
We should display a message asking the user to use the skill. Analogous to rating section.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #657 

Changes: Calculate total skill hits and display a message when skill is unused.

Surge Deployment Link: https://pr-661-fossasia-susi-skill-cms.surge.sh

Please suggest if the default message can be improved. 

Screenshots for the change:
image

### CODE REVIEW

**Comment by anshumanv** [comment]
Will squash commits after any needful changes

**Comment by anshumanv** [comment]
@akshatnitd resolved! :smile:

File: src/components/SkillUsageCard/SkillUsageCard.js [hunk 1]
```diff
@@ -4,23 +4,35 @@ import PropTypes from 'prop-types';
...
+						<LineChart width={600} height={300} data={this.props.skill_usage}
+								margin={{top: 5, right: 30, left: 20, bottom: 5}}>
+							<XAxis dataKey="date" padding={{right: 20}} />
+							<YAxis/>
+							<Tooltip wrapperStyle={{height: '60px'}}/>
+							<Legend />
+							<Line name='Skill usage count' type="monotone" dataKey="count" stroke="#82ca9d" activeDot={{r: 8}}/>
+						</LineChart>
+					</div>):
+					(<div className="default-message">No usage data available, try this skill now!</div>)
```
Comment (AnupKumarPanwar) [inline src/components/SkillUsageCard/SkillUsageCard.js]: Try this skill now doesn't look good here. As the user expects some button to try out the skill.

Comment (anshumanv) [reply src/components/SkillUsageCard/SkillUsageCard.js]: @akshatnitd is working on this already in https://github.com/fossasia/susi_skill_cms/issues/664 the same approach will then be applied to this section.

**Comment by anshumanv** [comment]
@prateekiiest that happened while resolving conflicts, fixed now.

**Pull request merged**